react/ConditionLayoutBinding.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useMemo } from 'react'
+import { useRuntime } from 'vtex.render-runtime'
+
+import ConditionLayout from './ConditionLayout'
+import type { NoUndefinedField, MatchType, Condition, Handlers } from './types'
+
+type Props = {
+  conditions: Array<Condition<ContextValues, HandlerArguments>>
+  matchType?: MatchType
+  Else?: React.ComponentType
+  Then?: React.ComponentType
+}
+
+type ContextValues = {
+  bindingId: string
+}
+
+type HandlerArguments = {
+  bindingId: { id: string }
+}
+
+const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
+  bindingId({ values, args }) {
+    return String(values.bindingId) === String(args?.id)
+  },
+}
+
+const ConditionLayoutBinding: StorefrontFunctionComponent<Props> = ({
+  Else,
+  Then,
+  matchType,
+  conditions,
+  children,
+}) => {
+
+  const { 
+    binding: {
+      id: bindingId
```

Comment:
Could this be `undefined`?

Reply:
I'm asking this because if `bindingId` is `undefined`, then the check here: https://github.com/vtex-apps/condition-layout/pull/39/files#diff-a2008f72585dee97b4df00dffe4f10851abde2eb8909ea836f1536dfa5558c68R48 could have false positives

Reply:
No, it cannot be undefined. Only strings allowed